src/main.py

```diff
@@ -18,11 +18,11 @@
 import gi
 import sys
 
-from gi.repository import Gtk, Gio, GLib
-from .window import GabtagWindow
-
 gi.require_version('Gtk', '3.0')
 
+from gi.repository import Gtk, Gio, GLib  # noqa: E402
```

Comment:
`# noqa: E402` is meant to be at the end of line 21 rather than at the end of line 23